top_section_styles.css

```diff
@@ -60,7 +66,7 @@ body {
 .word:hover:after {
     content: "";
     display: block;
-    width: 50px;
+    width: calc(100% - 10px);
```

Comment:
@IharDorinel i'm not sure that it will always work successfully when you calculate values with different types (% and px)